### ISSUE TITLE
[Bug]: Docker installation steps are not valid

### ISSUE THREAD

**Issue by jmaister** [LemmyNet/lemmy#3117]
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [X] Is this a UI / front end issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo.

### Summary

When following the steps on https://join-lemmy.org/docs/en/administration/install_docker.html to run lemmy with docker, it fails as the docker-compose.yml file requires the whole project to be cloned.

The  `lemmy` step requires to build the application. That means to clone the whole project instead of just doing wget on those two files.

https://github.com/LemmyNet/lemmy/blob/ebd8c86f0ee6e42f5b4f3212af7f7806c404b035/docker/docker-compose.yml#L41

### Steps to Reproduce

1. Follow instructions on https://join-lemmy.org/docs/en/administration/install_docker.html

### Technical Details

```
jordi@xxx:~/projects/lemmy$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
```

```
jordi@xxx:~/projects/lemmy$ docker --version
Docker version 20.10.21, build 20.10.21-0ubuntu1~20.04.2
```

### Version

main

**Comment by jhnhnck** [comment]
If you uncomment out line 38 and then comment out 41-43 it seems to work

**Comment by RyantHults** [comment]
there are a few additional updates that also need to be made. see https://github.com/LemmyNet/lemmy/pull/3172

**Comment by Nutomic** [comment]
I think this is fixed already, anyway it belongs in docs repo.

**Comment by jhnhnck** [comment]
I don't believe this is fixed as of 1.18.1-rc4, and it doesn't belong in the docs repo. 

This issue is related to the docker compose file in this repo not referencing the pre-built lemmy docker image, but instead requiring it be built from scratch which differs from how lemmy-ui and the other containers are currently defined. What the docs says would be correct if this file was fixed.

**Comment by Nutomic** [comment]
The docker-compose file in this repo is for development, not production. It also isnt linked in the current docs.
